core/Date.php

```diff
@@ -729,6 +731,10 @@ public function getLocalized($template)
             }
         }
 
+        if ($ucfirst) {
+          $out = Common::mb_strtoupper(mb_substr($out, 0, 1)) . mb_substr($out, 1);
```

Comment:
Is that really required? If I understand that correctly it should**n't** be the default to uppercase the first letter at all. The formattings from CLDR should define if the standalone version is used or not. And if a month name has a capital first letter or not is part of the translations. Or am I'm missing something?

Reply:
Ok. So guess your approach should be good enough for now. To solve the correctly guess we would need to fetch the context transformation info from CLDR (e.g. https://github.com/unicode-cldr/cldr-misc-full/blob/master/main/de/contextTransforms.json)

See http://cldr.unicode.org/translation/capitalization